test/gateway/ost_prime/increase_supply.js

```diff
@@ -141,18 +124,23 @@ contract('OSTPrime.increaseSupply()', function (accounts) {
       true,
       'Contract should return true.',
     );
+    let initialBeneficiaryBalance = await Utils.getBalance(beneficiary);
 
     await ostPrime.increaseSupply(
-      beneficiary,
-      amount,
-      { from: coGatewayAddress },
+        beneficiary,
+        amount,
+        {from: coGatewayAddress},
     );
 
-    let beneficiaryBalance = await ostPrime.balanceOf.call(beneficiary);
+    let finalBeneficiaryBalance = await Utils.getBalance(beneficiary);
+
+    initialBeneficiaryBalance = new BN(initialBeneficiaryBalance);
+    finalBeneficiaryBalance = new BN(finalBeneficiaryBalance);
```

Comment:
why convert to BN? Does `getBalance` return `String`? (I think I asked this before somewhere, sorry)
can we make `getBalance` return a `Number`?